tembo-cli/src/cmd/top.rs

```diff
@@ -0,0 +1,172 @@
+        format!("sum by(persistentvolumeclaim) (kubelet_volume_stats_capacity_bytes{{job=\"kubelet\", metrics_path=\"/metrics\", namespace=\"{}\"}}) - sum by(persistentvolumeclaim) (kubelet_volume_stats_available_bytes{{job=\"kubelet\", metrics_path=\"/metrics\", namespace=\"{}\"}})",namespace_encoded, namespace_encoded).to_string(),
+        format!("sum(container_memory_working_set_bytes{{job=\"kubelet\", metrics_path=\"/metrics/cadvisor\", namespace=\"{}\",container!=\"\", image!=\"\"}}) / sum(max by(pod) (kube_pod_container_resource_requests{{job=\"kube-state-metrics\", namespace=\"{}\", resource=\"memory\"}}))",namespace_encoded, namespace_encoded).to_string(),
+        format!("max by (pod) (cnpg_backends_max_tx_duration_seconds{{namespace=\"{}\"}})",namespace_encoded).to_string(),
+    ];
+
+            for query in &metric_queries {
+                //Looping it every 2 seconds to retrieve the response
+                match fetch_metric(query, &namespace_encoded, &client, &headers).await {
+                    Ok(metrics_response) => println!("{:?}", metrics_response),
+                    Err(e) => eprintln!("Error fetching metrics: {}", e),
+                }
+            }
+        }
+    }
+}
+
+async fn fetch_metric(
+    metric_query: &str,
+    namespace_encoded: &str,
+    client: &reqwest::Client,
+    headers: &HeaderMap,
+) -> Result<MetricsResponse> {
+    const BASE_URL: &str = "https://api.data-1.use1.tembo.io";
+    let query_encoded = urlencoding::encode(metric_query);
+    let url = format!(
+        "{}/{}/metrics/query?&query={}",
+        BASE_URL, namespace_encoded, query_encoded
+    );
+
+    //Sending the HTTP request with headers
+    let response = client
+        .get(&url)
+        .headers(headers.clone())
+        .send()
+        .await?
+        .json::<MetricsResponse>()
+        .await?;
+
+    Ok(response)
+}
+
+//Getting the org and instance name to run the queries
+async fn get_instance_org_name(
+    config: &Configuration,
+    env: &Environment,
+    instance_name: &String,
+) -> String {
+    let instance_id = get_instance_id(instance_name, &config, &env)
+        .await
+        .unwrap()
+        .unwrap();
+    let instance = get_instance(&config, &env.org_id.clone().unwrap(), &instance_id)
+        .await
+        .unwrap();
+    instance.organization_name
+}
+
+//Function to tackle async
+fn blocking(config: &Configuration, env: &Environment) -> Result<(), anyhow::Error> {
+    let rt = Runtime::new().unwrap();
+    let instance_settings = get_instance_settings(None, None)?;
+    rt.block_on(async {
+        let _metrics_response = fetch_metrics_loop(&config, env.clone(), instance_settings).await;
+    });
+    Ok(())
+}
+
+pub fn execute() -> Result<(), anyhow::Error> {
```

Comment:
If there is not a tembo.toml in the current directory, please provide an informative error message